systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -29,12 +32,13 @@
     function next_plan = getSuccessor(obj, t, x)
       if isnumeric(obj.qtraj)
         qf = obj.qtraj;
+        next_plan = QPLocomotionPlan.from_standing_state([x(1:6); qf(7:end); x(obj.robot.getNumPositions+1:end)], obj.robot, obj.supports(end));
+        next_plan.mu = obj.mu;
+        next_plan.g = obj.g;
       else
-        qf = fasteval(obj.qtraj, obj.qtraj.tspan(end));
+        % qf = fasteval(obj.qtraj, obj.qtraj.tspan(end));
+        next_plan = FrozenPlan(obj.last_qp_input);
```

Comment:
For manip plans, it's been better on the actual robot to just freeze the last output, rather than re-computing a new standing plan.